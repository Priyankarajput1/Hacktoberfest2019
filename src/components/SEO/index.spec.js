import React from 'react';
import renderer from 'react-test-renderer';

import SEO from './index';

describe('components/SEO', () => {
  it('should render', () => {
    const component = renderer.create(
      <SEO title="home" description="this is description" />
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
