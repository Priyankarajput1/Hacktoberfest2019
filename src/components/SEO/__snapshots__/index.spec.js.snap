// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/SEO should render 1`] = `null`;
